package.json

```diff
@@ -52,6 +53,8 @@
     "ember-fhir-adapter": "0.0.12",
     "ember-load-initializers": "0.5.1",
     "ember-modal-dialog": "0.8.3",
+    "ember-moment": "6.1.0",
```

Comment:
Yes, we still need it in Bower. `ember-moment` installs `ember-cli-moment-shim` in `package.json`, and `ember-cli-moment-shim` imports `moment.js` from our bower folder. Neither of them package `moment.js`, so we still need to include it.